test-cases/manager/manager-backup-restore-set-dataset.yaml

```diff
@@ -10,6 +10,14 @@ instance_type_db: 'i4i.xlarge'
 instance_type_loader: 'c6i.xlarge'
 
 region_name: us-east-1
+append_scylla_yaml:
+  auto_snapshot: false
+  object_storage_endpoints:
+    - name: s3.us-east-1.amazonaws.com
```

Comment:
if this would need to be enabled across longevity, we'll need a better way to configure it based  on `region_name`

Reply:
This configuration is now split to a separate yaml.
So in future we can simply add entries of regions to the list of `object_storage_endpoints`